dev.deedles.Trayscale.yml

```diff
@@ -0,0 +1,55 @@
+app-id: dev.deedles.Trayscale
+
+runtime: org.gnome.Platform
+runtime-version: '43'
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.golang
+
+command: trayscale
+
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+  - --share=network
+  - --share=ipc
+  - --filesystem=/run/tailscale:ro
+  - --talk-name=org.freedesktop.Notifications
+
+build-options:
+  env:
+    - GOROOT=/usr/lib/sdk/golang
+
+modules:
+  - name: trayscale
+    buildsystem: simple
+    build-commands:
+      - $GOROOT/bin/go build -v -trimpath -ldflags="-X 'deedles.dev/trayscale/internal/version.version=v0.7.12'" ./cmd/trayscale
+      - install -Dm0755 trayscale $FLATPAK_DEST/bin/trayscale
+      - install -Dm0644 dev.deedles.Trayscale.png $FLATPAK_DEST/share/icons/hicolor/256x256/apps/dev.deedles.Trayscale.png
+      - install -Dm0644 dev.deedles.Trayscale.desktop $FLATPAK_DEST/share/applications/dev.deedles.Trayscale.desktop
+      - install -Dm0644 dev.deedles.Trayscale.metainfo.xml $FLATPAK_DEST/share/metainfo/dev.deedles.Trayscale.metainfo.xml
+    sources:
+      - type: git
+        url: https://github.com/DeedleFake/trayscale
+        tag: v0.7.12
+        commit: 5bfd6c05e09e8b67d08441f403828a04c4ebb659
+      - type: archive
+        url: https://github.com/DeedleFake/trayscale/releases/download/v0.7.10/vendor.tar.gz
+        sha256: 1197f61a88e84a02366f7c5eac1bea0a7dea3cedbb831e1dfc3f02339bd537f8
+        strip-components: 0
+
+  - name: tailscale
+    buildsystem: simple
+    build-commands:
+      - install -Dm0755 tailscale $FLATPAK_DEST/bin/tailscale
+    sources:
+      - type: archive
+        only-arches: [x86_64]
+        url: https://pkgs.tailscale.com/stable/tailscale_1.36.2_amd64.tgz
+        sha256: 696d08ec8b9a73fe8f3cb537eaec93b45ee621d8bf0b7f5b954359a17ad185df
+      - type: archive
+        only-arches: [i386]
+        url: https://pkgs.tailscale.com/stable/tailscale_1.36.2_386.tgz
+        sha256: f874aef63a6e101b77e49c9e1a133f67f0b8849979f14561ae3dfe6912a41f64
```

Comment:
```suggestion
```

Won't support i386

However aarch64 is.....

Reply:
why not building it from source?

Reply:
I didn't want to have to bundle a `vendor` directory for their dependencies, but I guess it's probably less of a problem than trying to support every architecture of pre-built binaries. I've switched it to build it.

Reply:
there is that tool to generate a manifest for go dependencies.
https://github.com/flatpak/flatpak-builder-tools/tree/master/go-get

Reply:
I tried that originally but it didn't work because it requires that the initial build be done in GOPATH mode, and many newer modules don't work correctly in GOPATH mode, resulting in build failures. Vendoring makes it very easy to deal with, though, as I can just produce a bundle of all of the dependencies in a single archive. The only really awkward thing is trying to figure out where to host it.

Reply:
what works the best.